packages/common/src/store/premium-content/sagas.ts

```diff
@@ -449,17 +454,6 @@ function* handleTipGatedTracks(
   yield* call(updateGatedTracks, action.payload.userId, 'tip')
 }
 
-function* refreshPremiumTrackAccess(
```

Comment:
How come we no longer need to poll?

Reply:
we do still poll, just not polling via this function anymore